pkg/sql/sem/builtins/aggregate_builtins.go

```diff
@@ -744,9 +760,10 @@ func (a *concatAggregate) Result() (tree.Datum, error) {
 }
 
 // Reset implements tree.AggregateFunc interface.
-func (a *concatAggregate) Reset(context.Context) {
+func (a *concatAggregate) Reset(ctx context.Context) {
 	a.sawNonNull = false
 	a.result.Reset()
+	a.acc.Empty(ctx)
```

Comment:
This actually might be incorrect, will need to double check this tomorrow.

Reply:
The max aggregate doesn't reset its account here either